applications/StructuralMechanicsApplication/custom_elements/base_solid_element.h

```diff
@@ -447,6 +452,21 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION)  BaseSolidElement
     virtual void InitializeMaterial();
     
     /**
+     * Gives the StressMeasure used
+     */
+    virtual ConstitutiveLaw::StressMeasure GetStressMeasure() const;
+    
+    /**
+     * It updates the historical database
+     * @param rThisKinematicVariables: The kinematic variables to be calculated 
+     * @param PointNumber: The integration point considered
+     */ 
+    virtual void UpdateHystoricalDatabase(
```

Comment:
If it is not in the base class can not be call in the base class, which means I will duplicate a lot of code unncesarily